.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build RTSan Libs
+
+on:
+  push:
+    tags: ["v*"]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Download and Extract
+      run: make init
+
+    - name: Configure
+      run: make configure
+
+    - name: Build
+      run: make build
+
+    - name: Test
+      run: make test
+
+    - name: Upload artifact Ubuntu
+      if: matrix.os == 'ubuntu-latest'
+      uses: actions/upload-artifact@v4
+      with:
+        name: ubuntu_artifacts_rtsan
+        path: ./llvm-project/build/lib/linux/libclang_rt.rtsan-*.a
+        retention-days: 1
+
+    - name: Upload artifact Darwin
+      if: matrix.os == 'macos-latest'
+      uses: actions/upload-artifact@v4
+      with:
+        name: darwin_artifacts_rtsan
+        path: ./llvm-project/build/lib/darwin/libclang_rt.rtsan*.dylib
+        retention-days: 1
```

Comment:
Do these need to be separated? Can we achieve the same with a regex path? E.g.:

`path: ./llvm-project/build/lib/*/libclang_rt.rtsan*.*`

Reply:
Great point. Let me ponder and try it out

Reply:
Ok, I think this actually over-complicates things a bit. The issue being that later down the pipe we need to upload these assets (and name them something OS specific) and then download them.

Overall this way is a little less complex, despite the duplication.

Reply:
Sounds good, don't worry about it.